main.py

```diff
@@ -119,9 +129,23 @@ def draw_whitespace(drawing: Drawing):
 shapes = load_shapes()
 positions = ["top_left", "top_right", "bottom_left", "bottom_right"]
 
+subject_ID = input('Insert Subject ID')
+subject = Subject(subject_ID, date.today(), {})
+subject.create_data_template()
+
+
+start = 0
+while start == 0:
+    start = input('Ready to start the window? (1 - yes, 0 - no)')
+
 pygame.init()
 SCREEN = pygame.display.set_mode((800, 600))
 
+position_right = 0
+while position_right == 0:
```

Comment:
this doesn't work and I don't know why

Reply:
Like above, something like this should work

```python
while True:
    position_check = input('Is the game positioned correctly? (Type Y to confirm):\n')
    if position_check == 'Y':
        break
        
    # Quit the application? Restart the process? etc
```